src/payment/strategies/bolt/bolt-payment-initialize-options.ts

```diff
@@ -0,0 +1,7 @@
+export default interface BoltPaymentInitializeOptions {
+  /**
+   * When true, Bigcommerce's checkout will be used
+   * otherwise Bolt's full checkout take over will be assumed
```

Comment:
From a BC perspective, `useClientScript` isn't very self explanatory. Maybe name it something like  `useBigCommerceCheckout?: boolean`

Reply:
yeah, that seems more self explanatory